async/async-and-await/cs/readme.md

```diff
@@ -5,42 +5,35 @@ products:
 - dotnet-core
```

Comment:
This sample will appear under the ".NET Core" node in the samples browser. Currently, there is no option in the samples browser for .NET 5.0 and later, so nothing to do in the PR. Can you submit an internal request for that?

Here is an image of the current available options under the .NET node.

![image](https://user-images.githubusercontent.com/31348972/89981445-2d25f200-dc74-11ea-8d93-f44a0040a3c9.png)